practice-app/app/notification.py

```diff
@@ -3,89 +3,16 @@
 import urllib
 from datetime import datetime, timedelta
 from math import cos, asin, sqrt, pi
+from .dbinit import session, Notification
 
 notif_api = Blueprint('notif_api', __name__)
 API_KEY = "Google API Key"
 
-events = [
-    {
-        "eventId": 1,
-        "owner": "emre_gundogu",
-        "title":  "Tennis Game for Everyone",
-        "content": "People who are looking for a tennis court can apply this event. Our court is for you if you are seeking for a clean and well-lit court.",
-        "location": "Etiler Tennis Club",
-        "coordinates": (41.0869173, 29.0321301),
-        "date": "01.05.2021",
-        "hours": "15.00",
-        "sport": "Tennis",
-        "ageGroup": "Any age",
-        "skillLevel": "Any level",
-        "playerCapacity": "2",
-        "spectatorCapacity": "8",
-        "spectators": [{"username":"zulfikar_tunc"}],
-        "players": [{"username":"zulfikar_tunc"}]
-    }
-]
+@notif_api.route("/api/v1.0/notification",methods=["GET"])
+def get_notification(user_id):
+    notifications = session.query(Notification).filter(Notification.recipientID == user_id).all()
+    return jsonify(notifications), 200
```

Comment:
To properly serialize notifications:
```suggestion
    return {'notifications': [col.name: str(getattr(notification, col.name)) for col in notification.__table__.columns for notification in notifications]}, 200
```